src/utils/findPropertySource.test.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, it, expect } from 'vitest'
+import { findPropertySource } from './findPropertySource'
+
+describe('findPropertySource', () => {
```

Comment:
I think we generally avoid describe blocks so you can drop it and use individual `it` blocks as test cases for the `findPropertySource` function.

Reply:
I was looking at `getCleanUrl` for reference, which seems to be the only one using a `describe` block. 😅

Reply:
My bad then. I'm still normalizing the tests across the ecosystem. Thanks for adjusting this!